core/src/main/java/io/methvin/watcher/PathUtils.java

```diff
@@ -42,40 +43,36 @@ public static <T> SortedMap<Path, T> subMap(SortedMap<Path, T> pathMap, Path tre
     return pathMap.subMap(treeRoot, Paths.get(treeRoot.toString(), "" + Character.MAX_VALUE));
   }
 
-  public static SortedMap<Path, FileHash> createHashCodeMap(Path file, FileHasher fileHasher)
-      throws IOException {
-    return createHashCodeMap(Collections.singletonList(file), fileHasher);
-  }
-
-  public static SortedMap<Path, FileHash> createHashCodeMap(List<Path> files, FileHasher fileHasher)
-      throws IOException {
+  public static SortedMap<Path, FileHash> createHashCodeMap(
+      Path file, FileHasher fileHasher, FileTreeVisitor fileTreeVisitor) throws IOException {
     SortedMap<Path, FileHash> lastModifiedMap = new ConcurrentSkipListMap<>();
```

Comment:
maybe while you're at it you can change this map to be called `hashes` instead of `lastModifiedMap`. I think the name is wrong because previously this used the last modified time of the file rather than the hash.